cfme/common/provider.py

```diff
@@ -43,7 +43,7 @@ def all_types():
     return all_types
 
 
-class BaseProvider(Taggable, Updateable, SummaryMixin, Navigatable):
+class BaseProvider(WidgetasticTaggable, Updateable, SummaryMixin, Navigatable):
```

Comment:
we should make sure that middleware provider is properly updated before merging this. 
@psav, @hhovsepy ^^